lib/galaxy/webapps/galaxy/controllers/history.py

```diff
@@ -665,24 +679,25 @@ def view_multiple( self, trans, include_deleted_histories=False, order='update'
 
         # TODO: allow specifying user_id for admin?
         include_deleted_histories = galaxy.util.string_as_bool( include_deleted_histories )
-        order = order if order in ( 'update', 'name', 'size' ) else 'update'
+        order_by = self.history_manager.parse_order_by( order, default='update_time' )
+        limit = self._parse_int( limit, min=1 ) if limit != 'None' else None
```

Comment:
Since it's not actually a bug, and this works fine, I'm not going to hold up this PR for this comment.  I'll open a new PR implementing my first suggestion.